partcad/src/partcad/shape.py

```
@@ -27,22 +28,23 @@
 class Shape(ShapeConfiguration):
     name: str
     desc: str
+    requirements: dict | list | str
     svg_path: str
     svg_url: str
-    # shape: None | b3d.TopoDS_Shape | OCP.TopoDS.TopoDS_Solid
+    # shape: None | OCP.TopoDS.TopoDS_Solid
 
     errors: list[str]
 
     def __init__(self, config):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)